service/src/main/java/bio/terra/pipelines/app/controller/QuotasController.java

```diff
@@ -51,15 +51,17 @@ public ResponseEntity<ApiQuotaWithDetails> getQuotaForPipeline(String pipelineNa
     UserQuota userQuota =
         quotasService.getOrCreateQuotaForUserAndPipeline(
             user.getSubjectId(), validatedPipelineName);
+    String quotaUnits = quotasService.getPipelineQuota(validatedPipelineName).getQuotaUnits();
```

Comment:
could alternatively make a dedicated method just to retrieve the quota units from the db